lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
LOL hi @mattcipher !!

Click the "Unwatch" button in the upper right-hand corner of this PR.

http://lmgtfy.com/?q=how+do+i+unwatch+a+repo+on+github#seen

<img width="1051" alt="screen shot 2016-06-24 at 11 44 48 am" src="https://cloud.githubusercontent.com/assets/5606422/16342639/38c1b802-3a01-11e6-8316-16c6c76cded2.png">